abc.templates/deployments/contents/configs/README.md

```diff
@@ -0,0 +1,65 @@
+# Deployment Configurations
+
+### TLDR
+1. Create a `<repo-name>.yaml` file for each repository in `REPLACE_SUBDIRECTORY`.
+1. Customize the configuration.
+1. Build and deploy a new image with new configurations.
+
+### Configure Repository Access
+
+Each repository that needs to mint a token must be configured with an allowed set of permissions. Any request from a repository that is not configured will default to denying all access.
+
+Each configuration file can have 1 or more rules that it can match against and it will evaluate them in order, top down, until it finds a matching condition.
+
+The repository configuration file looks like below and is made up of three primary sections:
+
+```yaml
+# abcxyz/github-token-minter.yaml
+-
+  if: 'assertion.workflow_ref.startsWith("abcxyz/github-token-minter/.github/workflows/ci.yml")'
+  repositories:
+    - 'github-token-minter'
+  permissions:
+    issues: 'read'
+```
+
+The `if` clause uses the [Google Common Expression Language](https://github.com/google/cel-spec) to match an inbound OIDC token against a series of rules. Any attributes from the assertion object are available to match against and you can make this expression as simple or complicated as required.
+
+The object mirrors what is available in an OIDC token for a GitHub Workflow, below is an example token.
```

Comment:
I would add a link, as this changes over time: 
https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/about-security-hardening-with-openid-connect#configuring-the-oidc-trust-with-the-cloud